portal-impl/src/com/liferay/portlet/documentlibrary/service.xml

```diff
@@ -136,7 +136,7 @@
 			<finder-column name="custom2ImageId" />
 		</finder>
 	</entity>
-	<entity human-name="document library file entry metadata" local-service="true" name="DLFileEntryMetadata" remote-service="false" uuid="true">
+	<entity external-reference-code="company" human-name="document library file entry metadata" local-service="true" name="DLFileEntryMetadata" remote-service="false" uuid="true">
```

Comment:
Shouldn't these be scoped by group? Even though it doesn't have a group ID column, it depends on a file entry/ddm structure (which are group scoped).

Reply:
Note that if this is a service builder restriction (i.e. setting it as group-scoped ERC would require the entity to have a group ID column) it is fine to leave it as company scoped.

Reply:
Hi

this is a restriction, the table does not have group column so must be on company scope

<img src="https://media3.giphy.com/media/3o6nUWTj5OVcAAZSfK/giphy.gif"/>